org.civicrm.reqangular/src/common/angular-date/src/services/DateFormatFactory.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * This factory provides access to date format from CRM settings
+ * @param $q
+ * @returns {{dateFormat: null, getDateFormat: getDateFormat}}
+ * @constructor
+ */
+function DateFormatFactory($q) {
+    return {
+        /**
+         * keeps information about date format
+         */
+        dateFormat: null,
```

Comment:
:facepalm:

right and I was right back then cause otherwise you cannot check if you have already cached the value. Apologies